entities/index.ts

```diff
@@ -2,13 +2,13 @@ export * from './Workspace';
 export * from './BusinessUnit';
 export * from './Schema';
 export * from './common/Field';
-export * from './Purpose';
+export * from './Purpose/Purpose';
 export * from './Connector';
 export * from './Event';
 export * from './Action';
 export * from './Segment';
-export * from './indicators/ActivityIndicator';
-export * from './indicators/CalculatedIndicator';
+export * from './Indicator/ActivityIndicator';
+export * from './Indicator/CalculatedIndicator';
```

Comment:
rename directory back to `indicators` - it's grouping of several actual config-entities

Reply:
so when you group config-entities or anything not model/model-like then you keep to lowercase? We reserve uppercase directories for grouping of Model/Class entities?